pom.xml

```diff
@@ -409,13 +410,22 @@
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maves-surefire-plugin</artifactId>
+                    <artifactId>maven-surefire-plugin</artifactId>
                     <version>${maven-surefire-plugin-version}</version>
+                    <configuration>
+                        <includes>
+                            <include>**/*Test.java</include>
+                        </includes>
+                        <excludes>
+                            <exclude>**/*IT.java</exclude>
+                            <exclude>**/*IntegrationTest.java</exclude>
+                        </excludes>
+                    </configuration>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-failsafe-plugin</artifactId>
-                    <version>${maven-surefire-plugin-version}</version>
+                    <version>${maven-failsafe-plugin-version}</version>
```

Comment:
This is OK.

Reply:
The versions are the same today, but that may not always be the case. Or, there may be a bug in one of the plugins that requires reverting to a previous version. Finally, just for clarity, the version property name should match the artifact ID.

Reply:
Yeah, I think this is OK.